### PR TITLE
Add comment about ulimit format.

### LINKED ISSUE #12116: Issue on the new ulimit feature of the run command 1.6 RC4
This is minor and probably won't be caught for some time but... 
See code command snippet

```
vagrant@ubuntu-1:~$ docker -v
Docker version 1.6.0-rc4, build e2e39fc
vagrant@ubuntu-1:~$ cid=$(docker run -itd --ulimit data=8192 ubuntu)
vagrant@ubuntu-1:~$ docker logs $cid
bash: xmalloc: .././locale.c:81: cannot allocate 2 bytes (0 bytes allocated)
```

### PR DESCRIPTION
**Pull request by calavera**
Fixes #12116

/cc @cpuguy83

Signed-off-by: David Calavera david.calavera@gmail.com

### CODE REVIEW

**Comment by cpuguy83** [comment]
LGTM
We should add an intermediate issue for someone to do this conversion.

File: docs/sources/reference/commandline/cli.md [hunk 1]
```diff
@@ -2374,6 +2374,9 @@ set on the daemon.
 > `as` option is disabled now. In other words, the following script is not supported:
 >   `$ docker run -it --ulimit as=1024 fedora /bin/bash`
 
+The values are send to the appropriate syscall as they are set, Docker doesn't
```
Comment (moxiegirl) [inline docs/sources/reference/commandline/cli.md]: The values are sent to the appropriate `syscall` as they are set. Docker doesn't perform any byte conversion. Take this into account when setting the values.

**Comment by calavera** [comment]
Fixed, thanks @moxiegirl

**Comment by thaJeztah** [comment]
Thanks! LGTM

**Pull request merged**